test/parallel/test-net-finished.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+const common = require('../common');
+const net = require('net');
+const { finished } = require('stream');
+
+const server = net.createServer(function(con) {
+  con.on('error', common.mustCall());
```

Comment:
```suggestion
  con.on('close', common.mustCall());
```
Isn't this the intended behaviour because you are closing the server at line 20?